package.json

```diff
@@ -62,8 +65,8 @@
     "release:minor": "env SEMANTIC=minor npm run release",
     "release:patch": "env SEMANTIC=patch npm run release",
     "release": "npm version ${SEMANTIC:-patch} -m \"Release %s\" && git push && git push --tags && npm publish",
-    "start:production": "node ./lib/server.js",
-    "start": "bash -c 'source env.sh; env UPPYSERVER_DATADIR=./output UPPYSERVER_DOMAIN=localhost:3020 UPPYSERVER_PROTOCOL=http UPPY_ENDPOINT=localhost:3452 nodemon ./src/server.js --exec babel-node'",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start:production": "node ./lib/start-standalone.js",
+    "start": "bash -c 'source env.sh; env UPPYSERVER_DATADIR=./output UPPYSERVER_DOMAIN=localhost:3020 UPPYSERVER_PROTOCOL=http UPPY_ENDPOINT=localhost:3452 nodemon ./src/start-standalone.js --exec babel-node'",
+    "test": "npm run lint && env UPPYSERVER_DATADIR=./test/output jest --config=test/conf.json"
```

Comment:
I'd probably move the contents of `conf.json` to the `jest` property in `package.json`. You can then run `jest` without config.